tests/maf/test_3x2fom.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+import os
+import tempfile
+import shutil
+from rubin_sim.data import get_data_dir
+from rubin_sim.utils.CodeUtilities import sims_clean_up
+import rubin_sim.maf as maf
+
+
+class Test3x2(unittest.TestCase):
+
+    @classmethod
+    def tearDownClass(cls):
+        sims_clean_up()
+
+    def setUp(self):
+        self.outDir = tempfile.mkdtemp(prefix='TMB')
+
+    @unittest.skipUnless(os.path.isdir(os.path.join(get_data_dir(), 'maps')),
+                 "Skipping 3x3 metric test because no dust maps.")
+    def test_3x2(self):
+        bundleList = []
+        nside = 64
+        colmap = maf.batches.colMapDict.ColMapDict('fbs')
+        nfilters_needed = 6
+        lim_ebv = 0.2
+        ptsrc_lim_mag_i_band = 25.9
+        m = maf.metrics.ExgalM5_with_cuts(m5Col=colmap['fiveSigmaDepth'], filterCol=colmap['filter'],
```

Comment:
I like running it this way since this is how it's run in the science batch, and ideally everything in the science batch would have a unit test that's good enough that it won't surprise crash like it did this time.